port_ocean/core/integrations/mixins/sync_raw.py

```diff
@@ -209,6 +195,37 @@ async def _register_in_batches(
         )
         return entities, errors
 
+    async def _search_deleted_entities(
```

Comment:
you are not searching deleted entities, you are searching entities to delete

Reply:
I dont agree with that either
Nothing in this function says delete